### ISSUE TITLE
Automated builds

### ISSUE THREAD

**Issue by sneedium** [Sneederix/SneedMC#3]
We (s)need a sneedful way to automatically build stuff. Preferrably both on commits and on tags.

No Github actions allowed, not sneedful (closed source soyware). I was thinking something like Travis or another self hosted CI on sneedmc.org

**Comment by sneedium** [comment]
Also AUR and MPR are retarded, but we have to use them to have a wide userbase. Don't care enough not to use them.

probably will be using COPR too because i don't want to unironically use fedora

**Comment by sneedium** [comment]
Let's not forget Open/FreeBaSeD, if we use a sneedful CI with master-slave arch, then we can figure out a way to build there. I don't have autism so I don't know how to use BSD we'll figure that out later

Slackware is a meme, unless some rando gives me a slackbuild i'm not supporting it

**Comment by binex-dsk** [comment]
i have infrastructure available that's used by PolyMC currently, using Jenkins. It does automatic pushes to the AUR and MPR, and can just use NGINX static directory serving.

Because I already have everything available, you can point everything to my home server; using `jenkins.sneedmc.org`, `packages.sneedmc.org`, etc. Meta can also easily be mirrored from PolyMC

PolyMC's Jenkins script also starts a COPR build, makes an AppImage, Windows build, and more cool stuff.

**Comment by sneedium** [comment]
BASED

If your VPS ip is the same as your website swurl.xyz then i can set DNS up and talk to you about sneeding stuff up later

**Comment by binex-dsk** [comment]
No, those are hosted on my home server, not my VPS. You can see the IP in [my-ip](https://github.com/binex-dsk/my-ip)

**Comment by medicinalcocaine3434** [comment]
@binex-dsk @sneedium blatant samefaggotry

**Comment by binex-dsk** [comment]
@medicinalcocaine3434 Cool it with the anti-semitism.

**Comment by LennyMcLennington** [comment]
> @binex-dsk @sneedium blatant samefaggotry

*BASED schizophrenic talking to himself... AWESOME!
